### PR TITLE
fix missing srollbar in long filter list

### PR DESCRIPTION
**Pull request by 0xF013**
[`* `#16548` missing srollbar in long filter list`](http://community.openproject.org/work_packages/16548)

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://0xf013-feature-16548-missing-scrollbar-in-long-filter-list.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

**Comment by myabc** [comment]
:+1:

File: app/assets/stylesheets/content/_work_packages_filters.sass [hunk 1]
```diff
@@ -2,7 +2,8 @@
   background-color: #efefef
   border:   1px solid #E6E6E6
   padding:  10px
-
+  max-height: 250px
+  overflow-y: auto
```
Comment (myabc) [inline app/assets/stylesheets/content/_work_packages_filters.sass:6]: @0xF013 (for next time) please keep the empty line. I think it makes it more readable.

**Comment by 0xF013** [comment]
@myabc ok, I think it was the autoformatter

**Pull request merged**